register/models.py

```diff
@@ -87,8 +95,11 @@ def confirm(self, cancellation_url):
             self.status = 'C'
             self.save()
 
+    def can_be_cancelled(self):
+        return self.status == 'C' or self.status == 'I'
```

Comment:
Maybe now we should use the constants APP_CONFIRMED, APP_INVITED, etc. instead of their 'magic' value?